targets/ChibiOS/_common/targetHAL_ConfigurationManager.cpp

```
@@ -121,6 +121,52 @@ __nfweak void ConfigurationManager_EnumerateConfigurationBlocks()
+            else if (fallbackNetworkConfigs != NULL)
+            {
+                platform_free(fallbackNetworkConfigs);
+            }
+        }
+
+        // Apply the same synthetic fallback to wireless settings.
+        if (networkWirelessConfigs->Count == 0)
+        {
+            static HAL_Configuration_Wireless80211 fallbackWirelessConfig;
+
+            HAL_CONFIGURATION_NETWORK_WIRELESS80211 *fallbackWirelessConfigs =
```

Clang (14.0.6)

[warning] use auto when initializing with a cast to avoid duplicating the type name

(modernize-use-auto)